### ISSUE TITLE
Linux ALVR (in docker container, with GPU)

### ISSUE THREAD

**Issue by ayunami2000** [alvr-org/ALVR#1085]
## Description

So I somehow got it this far with my setup, but unfortunately, it dies here:
![image](https://user-images.githubusercontent.com/24785136/172274950-5d51f69f-16f6-42a6-a816-6272c9da7176.png)
![image](https://user-images.githubusercontent.com/24785136/172274975-bf42f4bf-2ff1-4966-83d5-1f3e17b4d28c.png)

Docker container being used: `ghcr.io/ehfd/nvidia-glx-desktop:20.04`

Note that it works all the way up to when I actually connect the VR headset, THEN it actually closes. But then, sometimes it just dies here:
![image](https://user-images.githubusercontent.com/24785136/172276138-0b373904-22b0-4e09-ae4a-16b8b3cea0c3.png)

The GUI works (in firefox because chrome requires --no-sandbox and well ALVR doesn't provide that sooooooo, i specified the binary to an empty `.sh` script for ALCRO_BROWSER_PATH) but the GUI closes once it says ServerQuitting

OK i figured it out (sorry if this is hard to follow), if I launch it with the ALVR icon, it will actually open the SteamVR piece. But if I run it via simply running SteamVR in steam, it will error like above.

![image](https://user-images.githubusercontent.com/24785136/172276545-3a4b1619-a5e7-4fe0-be95-e567647f498d.png)

Ok so now I am at a place where ALVR is loaded (in firefox) and SteamVR is showing this. NOW when I actually open ALVR on my Quest 2, this happens:
![image](https://user-images.githubusercontent.com/24785136/172276753-a2ec41f3-5c21-498d-b418-e9a4e5738c6e.png)

There are SO MANY `vr` log files, so please let me know which ones to look at lmfao.

## General Troubleshooting
- [x] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
- [x] I read the ALVR Wikis [here](https://github.com/polygraphene/ALVR/wiki) and [here](https://github.com/alvr-org/ALVR/wiki)

### Hardware
**Note**: for Linux, an upload to the [`hw-probe`](https://linux-hardware.org/) database is preferred: `hw-probe -all -upload`
https://linux-hardware.org/?probe=3ee16e0227

**CPU**:
see hw-probe results
**GPU**:
see hw-probe results
**Audio**:
see hw-probe results
### Installation
**ALVR Version**:
latest as of when this issue was created on the ALVR ubuntu repository
**SteamVR Version**:
SteamVR 1.22.13
**Install Type**:
- [x] Packaged (`exe`, `deb`, `rpm`, etc)
- [ ] Portable (`zip`)
- [ ] Source

**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions):
Ubuntu 20.04.4 LTS

**Comment by ayunami2000** [comment]
Alright, I fixed the issue with the output (It was lack of chrome). Now, the only remaining issue is that SteamVR still throws an error + the error 303 in SteamVR

**Comment by pixel844** [comment]
> Alright, I fixed the issue with the output (It was lack of chrome). Now, the only remaining issue is that SteamVR still throws an error + the error 303 in SteamVR

im having the same problem

**Comment by ayunami2000** [comment]
UPDATE: I've managed to get past the `CEncoder::Run` issue (`XDG_RUNTIME_DIR` was not set!), but now it's stuck on `CEncoder: Listening`. And SteamVR still shows the 303 error.

**Comment by ayunami2000** [comment]
> @ayunami2000 how did yo replace vrcompositor-launcher? I am gettig this two now

I replaced it with a bash script that launched vrcompositor if I remember correctly.

**Comment by guysoft** [comment]
Looks like it set here but still getting errors, will investigate. thanks though. If you have any tips would be great
